packages/server-wallet/src/__test__/create-and-close-channel/fake-funding.test.ts

```diff
@@ -83,20 +83,20 @@ it('Create a fake-funded channel between two wallets ', async () => {
     turnNum: 0,
   });
 
-  // after joinChannel, B double-signs PreFund0
+  // after joinChannel, B signs PreFund1
   const bJoinChannelOutput = await b.joinChannel({channelId});
   expect(getChannelResultFor(channelId, [bJoinChannelOutput.channelResult])).toMatchObject({
     status: 'opening',
-    turnNum: 0,
+    turnNum: 1,
   });
 
-  // B sends countersigned PreFund0 to A
+  // B sends signed PreFund1 to A
   const aPushJoinChannelOutput = await a.pushMessage(
     getPayloadFor(participantA.participantId, bJoinChannelOutput.outbox)
   );
   expect(getChannelResultFor(channelId, aPushJoinChannelOutput.channelResults)).toMatchObject({
-    status: 'opening',
-    turnNum: 0,
+    status: 'running',
+    turnNum: 2,
```

Comment:
If this test passes, it looks like a bug. If the turn number is `2`, the channel shouldn't be `running`?

Reply:
Currently PostFundSetup counts as 'running' I believe https://github.com/statechannels/statechannels/blob/02a2b0d0cff63f1fadfd4ca22bba31671c17dfa5/packages/server-wallet/src/protocols/state.ts#L76

Reply:
Should that be changed?

Reply:
Yes but maybe not in the scope of this PR?